api/users/serializers.py

```diff
@@ -66,7 +70,10 @@ def update(self, instance, validated_data):
                     instance.social[key] = val
             else:
                 setattr(instance, attr, value)
-        instance.save()
+        try:
+            instance.save()
+        except ValidationValueError as e:
+            raise InvalidModelValueError(detail=e.message)
```

Comment:
Can you ensure that a default value is passed here? `e.message or None` would likely work.

If I'm reading this right: if any model raises a bare ValidationValueError (no message provided), then `e.message` will be a blank string. Since `detail` is only ignored if it is `None`, an empty ValidationValueError (a blank string) would result in an unhelpful error message to the user.

See:
> site-packages/rest_framework/exceptions.py#41

And example:
```python
try: 
    raise ValidationValueError
except Exception as e:
    print type(e.message)
```    
Gives 
```<type 'str'>```